prow/e2e-simpleTests-cni.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright 2018 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#######################################
+#                                     #
+#         e2e-simpleTest-cni          #
+#                                     #
+#######################################
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+echo 'Running e2e_simple test with rbac, auth Tests and CNI enabled'
+export ENABLE_ISTIO_CNI=true
+# cniBinDir setting is appropriate for GKE environments
+export EXTRA_HELM_SETTINGS="--set istio-cni.excludeNamespaces={} --set istio-cni.cniBinDir=/home/kubernetes/bin --set istio-cni.tag=nightly-master"
```

Comment:
If the istio/release daily builds istio-cni, this needs to set `istio_cni.hub=gcr.io/istio-release` and `istio_cni.tag=master-latest-daily`

Reply:
@tiswanso - I should remove the hub and tag setting entirely so that the hub and tag are based on what is checked into to the CNI repo.